src/test/java/address/testutil/TestUtil.java

```diff
@@ -10,19 +10,26 @@
 import address.sync.cloud.model.CloudAddressBook;
 import address.sync.cloud.model.CloudPerson;
 import address.sync.cloud.model.CloudTag;
+import com.google.common.io.Files;
```

Comment:
com.google.guava

yea, better than java File Util. Does all the dirty jobs for u.